app/scripts/modules/core/src/presentation/main.less

```diff
@@ -289,9 +289,9 @@ html {
     position: fixed;
     margin-top: -20px;
     right: 0;
-    height: 100%;
+    height: 98%;
```

Comment:
this seems odd, why?

Reply:
I noticed that the width at 100% was pushing the drawer out of the screen height, which is why the scrolling issues were cropping up. This was more dramatic from a projects view or with a custom header.